runtime/drivers/postgres/postgres.go

```diff
@@ -56,6 +56,10 @@ var spec = drivers.Spec{
 
 type driver struct{}
 
+type ConfigProperties struct {
+	DatabaseURL string `mapstructure:"database_url"`
```

Comment:
We are in this weird situation where everywhere else it is called a DSN but called `database_url` in postgres because it the more popular word in postgres world. May be we keep both and let user specify any one ?

Reply:
Yeah supporting both and aliasing one as the other seems good to me